interactive_engine/compiler/src/main/java/com/alibaba/graphscope/gremlin/plugin/processor/IrStandardOpProcessor.java

```diff
@@ -138,6 +147,8 @@ public IrStandardOpProcessor(
         this.idGenerator = idGenerator;
         this.queryCache = queryCache;
         this.executionClient = executionClient;
+        this.printThreshold = Long.parseLong(configs.get("query.print.threshold.ms", "200"));
```

Comment:
default to print all to be consistent to previous semantic. People maybe confused that some query are not printed if they're just started to try it out.